client/app/telecom/telephony/line/calls/abbreviatedNumbers/telecom-telephony-line-calls-abbreviatedNumbers.controller.js

```diff
@@ -9,21 +9,22 @@ angular.module('managerApp').controller('TelecomTelephonyLineCallsAbbreviatedNum
   }
 
   remove({ abbreviatedNumber }) {
-    return this.OvhApiTelephony.AbbreviatedNumber().v6().remove({
+    return this.OvhApiTelephony.Line().AbbreviatedNumber().v6().remove({
       billingAccount: this.$stateParams.billingAccount,
+      serviceName: this.$stateParams.serviceName,
       abbreviatedNumber,
     }).$promise;
   }
 
   insert(abbreviatedNumber) {
-    return this.OvhApiTelephony.AbbreviatedNumber().v6().insert({
+    return this.OvhApiTelephony.Line().AbbreviatedNumber().v6().insert({
       billingAccount: this.$stateParams.billingAccount,
       serviceName: this.$stateParams.serviceName,
     }, abbreviatedNumber).$promise;
   }
 
   update(abbreviatedNumber) {
```

Comment:
You could use destructive pattern like you did above with remove method

Reply:
Do you mean destructuring assignment pattern for the method argument?

Reply:
Yes i mean it